controllers/submariner/cleanup.go

```diff
@@ -120,6 +120,29 @@ func (r *Reconciler) ensureServiceDiscoveryDeleted(ctx context.Context, namespac
 	return true
 }
 
+func (r *Reconciler) ensureGlobalnetDaemonSetDeleted(ctx context.Context, instance *operatorv1alpha1.Submariner) bool {
```

Comment:
The return value isn't used.

Reply:
@tpantelis Removed return value. See commit https://github.com/submariner-io/submariner-operator/pull/2027/commits/565529a638b7c43c356830f01258bb874b3d4b85